### PR TITLE
PR for Cruise Control(CC) Artifacts

### PR DESCRIPTION
**Pull request by npecka**
### Type of change

- Enhancement / new feature

### Description

Changed jcenter link to linkedin.jfrog link due to depreciation of jcenter

Resolves --> https://github.com/strimzi/strimzi-kafka-operator/issues/4587 

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [x] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md
- [ ] Supply screenshots for visual changes, such as Grafana dashboards

### CODE REVIEW

File: docker-images/kafka/kafka-thirdparty-libs/2.6.x/pom.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
     <repositories>
         <repository>
             <id>jcenter</id>
```
Comment (scholzj) [inline docker-images/kafka/kafka-thirdparty-libs/2.6.x/pom.xml:26]: Maybe you can rename the `id` from `jcenter` to `cruisecontrol` or something like that to correspond to the repository?

File: docker-images/kafka/kafka-thirdparty-libs/2.7.x/pom.xml [hunk 2]
```diff
@@ -24,7 +24,7 @@
     <repositories>
         <repository>
             <id>jcenter</id>
```
Comment (scholzj) [inline docker-images/kafka/kafka-thirdparty-libs/2.7.x/pom.xml:26]: Same as above

File: docker-images/kafka/kafka-thirdparty-libs/cc/pom.xml [hunk 3]
```diff
@@ -22,7 +22,7 @@
     <repositories>
       <repository>
         <id>jcenter</id>
```
Comment (scholzj) [inline docker-images/kafka/kafka-thirdparty-libs/cc/pom.xml:24]: Same as above

**Review by scholzj — COMMENTED** [review]
Thanks for the PR. I left few nits, but LGTM otherwise.

File: docker-images/kafka/kafka-thirdparty-libs/2.6.x/pom.xml (hunk 1 above)
Comment (npecka) [reply docker-images/kafka/kafka-thirdparty-libs/2.6.x/pom.xml]: I've made the changes and pushed in new commit. Thanks!

File: docker-images/kafka/kafka-thirdparty-libs/2.7.x/pom.xml (hunk 2 above)
Comment (npecka) [reply docker-images/kafka/kafka-thirdparty-libs/2.7.x/pom.xml]: I've made the changes and pushed in new commit. Thanks!

File: docker-images/kafka/kafka-thirdparty-libs/cc/pom.xml (hunk 3 above)
Comment (npecka) [reply docker-images/kafka/kafka-thirdparty-libs/cc/pom.xml]: I've made the changes and pushed in new commit. Thanks!

**Review by scholzj — APPROVED** [review]

**Comment by scholzj** [comment]
@npecka Could you please fix the DCO signoff? The details should normally be under the _Details_ link next tot he DCO status.

**Comment by scholzj** [comment]
@kyguy @strimzi/maintainers Can we get some reviews here?

**Review by kyguy — APPROVED** [review]
Thanks for the fix @npecka, looks great!

**Review by ppatierno — APPROVED** [review]

**Pull request merged**